.gitignore

```diff
@@ -8,3 +8,4 @@ composer.phar
 .buildpath
 .project
 .settings
+composer.lock
```

Comment:
Why would you *not* track the lock file? If there are dependencies on a specific version of some other lib, then surely the lock file should ensure that this lib has that dependency available to it. Otherwise updating the dependencies might break this library.